src/transformers/expect-js.test.js

```diff
@@ -0,0 +1,291 @@
+/* eslint-env jest */
+import chalk from 'chalk';
+import { wrapPlugin } from '../utils/test-helpers';
+import plugin from './expect-js';
+
+chalk.enabled = false;
+const wrappedPlugin = wrapPlugin(plugin);
+
+function testChanged(msg, source, expectedOutput, options = {}) {
+    test(msg, () => {
+        const result = wrappedPlugin(source, options);
+        expect(result).toBe(expectedOutput);
+
+        // Running it twice should yield same result
+        expect(wrappedPlugin(result, options)).toBe(result);
+    });
+}
+
+testChanged(
+    'does not touch code without expect require/import',
+    `
+    const test = require("testlib");
+    test(t => {
+      expect(stuff).toExist();
```

Comment:
It seems expect.js doesn't have `toExist`, so for this test to be more valuable it would be nice to change this to `to.be.ok();` (that expect.js) supports